src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -2450,8 +2450,8 @@ namespace FanCoilUnits {
                             Calc4PipeFanCoil(state, FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOut);
                         }
                     }
-                    QUnitOut = AirMassFlow *
-                               (PsyHFnTdbW(Node(OutletNode).Temp, Node(InletNode).HumRat) - PsyHFnTdbW(Node(InletNode).Temp, Node(InletNode).HumRat));
+                    QUnitOut = AirMassFlow * Psychrometrics::PsyDeltaHSenFnTdb2W2Tdb1W1(
+                                                 Node(OutletNode).Temp, Node(InletNode).HumRat, Node(InletNode).Temp, Node(InletNode).HumRat);
```

Comment:
I was expecting this to be one of the new `Calc...` functions, namely CalcZoneSensibleOutput`. Here and many other places in FanCoilUnits, HVACMultiSpeedHeatPump, HVACSingleDuct, and UnitarySystem.

Reply:
I used CalcZoneSensibleOutput in "Corrector" only. I'm not sure if we want to use in other places without investigating its implication.

Reply:
I think the FanCoil unit can use this new function (CalcZoneSensibleOutput) but other places have to be investigated case by case.

Reply:
Updated FanCoilUnit to use the new function CalcZoneSensibleOutput() in several places.